components/Calendar/Day.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { css } from 'glamor'
+import { fontStyles, useColorContext } from '@project-r/styleguide'
+import { columnDateFormat, reformatUrlDate } from './utils'
+import Repo from './Repo'
+import { ascending, group } from 'd3-array'
+import withT from '../../lib/withT'
+
+const styles = {
+  container: css({
+    flex: '1 1 0',
+    display: 'flex',
+    flexDirection: 'column'
+  }),
+  reposContainer: css({
+    borderLeftWidth: 1,
+    borderLeftStyle: 'solid',
+    flexGrow: 1
+  }),
+  dateHeading: css({
+    display: 'block',
+    marginBottom: 10,
+    ...fontStyles.sansSerifMedium14
+  }),
+  templateHeading: css({
+    display: 'block',
+    marginBottom: 10,
+    paddingLeft: 10,
+    ...fontStyles.sansSerifRegular14
+  }),
+  templateContainer: css({
+    paddingBottom: 20
+  })
+}
+
+const TemplateHeading = withT(({ t, template }) => (
+  <span {...styles.templateHeading}>{t(`repo/calendar/${template}`)}</span>
+))
+
+const ReposByTemplate = ({ template, repos, ...props }) =>
+  repos ? (
+    <div {...styles.templateContainer}>
+      <TemplateHeading template={template} />
+      {repos
+        .sort((repo1, repo2) =>
+          ascending(
+            new Date(repo1.meta.publishDate),
+            new Date(repo2.meta.publishDate)
+          )
+        )
+        .map(repo => (
+          <Repo key={repo.id} repo={repo} {...props} />
+        ))}
+    </div>
+  ) : null
+
+const DateHeading = ({ date }) => (
+  <span {...styles.dateHeading}>{reformatUrlDate(date, columnDateFormat)}</span>
+)
+
+const Day = ({ day: { date, repos }, isPast }) => {
+  const [colorScheme] = useColorContext()
+  const reposByTemplate = group(repos, repo =>
+    repo.latestCommit.document.meta.template === 'editorialNewsletter'
+      ? 'newsletter'
+      : 'other'
+  )
+  return (
+    <div
+      {...styles.container}
+      {...colorScheme.set('color', isPast ? 'textSoft' : 'text')}
```

Comment:
question: wouldn't it be easier to just set an opacity: 0.6 (or so) on the entire day? Also, it seems like in the current version, dates are still changeable on past repos, is that intentional?

Reply:
I believe things can be republished – if you add a correcting for instance or if you make edits to some pages. Can check with production though.